lib/clickhousex/protocol.ex

```diff
@@ -18,7 +18,7 @@ defmodule Clickhousex.Protocol do
   @type result :: Clickhousex.Result.t()
   @type cursor :: any
 
-  @ping_query Clickhousex.Query.new("SELECT 1") |> DBConnection.Query.parse([])
```

Comment:
`Clickhousex.Query.new` already calls `DBConnection.Query.parse(statement, [])`, calling it second time is no-op.